src/app/clusters/ota-requestor/OTARequestor.h

```diff
@@ -31,6 +31,10 @@
 
 #include "BDXDownloader.h"
 
+#ifdef OTA_PERIODIC_TIMEOUT
```

Comment:
We shouldn't be modifying this file. The change should go in the EFR files and we'll call the OTA API from there (will probably do it in a separate PR)